components/normalizer/benches/data/TestNames_Japanese_h.txt

```diff
@@ -0,0 +1,59 @@
+#/**
+# * © 2023 and later: Unicode, Inc. and others.
+# * License & terms of use: http://www.unicode.org/copyright.html
+# *******************************************************************************
+# * Copyright (C) 2002-2023, International Business Machines Corporation and    *
+# * others. All Rights Reserved.                                                *
+# *******************************************************************************
+# */
```

Comment:
Here and elsewhere: Instead of this license header, please use the standard ICU4X header, which is compatible with it.

```
# This file is part of ICU4X. For terms of use, please see the file
# called LICENSE at the top level of the ICU4X source tree
# (online at: https://github.com/unicode-org/icu4x/blob/main/LICENSE ).
```

Reply:
Done !